src/js/shield_defs.js

```diff
@@ -3856,6 +3863,9 @@ export function loadShields() {
     },
   };
 
+  shields["US:NY:Inner_Loop"].refsByName = {
+    "Inner Loop": "LOOP",
+  };
```

Comment:
This works, but I wonder now if we should add a function to hard-code the ref based on network for cases like this.

Reply:
There’s only ever going to be one route in the `US:NY:Inner_Loop` network, right? It doesn’t need to be a function; we could add support for setting a static `ref` property in the shield definition and using that instead of `refsByName`.

Reply:
#1070 adds this support

Reply:
New functionality integrated in 1b3799d